content/docs/1.0.0/tutorial/deploy-your-microservice.md

```diff
@@ -124,12 +203,13 @@ We can also open up the [Freeboard dashboard](http://localhost:10102/?load=dashb
 
 Once a microservice is up and running you will probably need to modify it later, which we can also do with the Controller.
 
-The `microservice update` command is used to update a particular microservice:
+The `microservice update` command is used to update a particular microservice. We can easily update the configuration of our moving average microservice.
```

Comment:
If we're staying consistent with your feedback, **Moving Average** should be capitalized here.